### PR TITLE
Make horizontal offset configurable

### PR DESCRIPTION
**Pull request by rubenvanerk**
Solves the issue as described in this comment: https://github.com/smalot/pdfparser/issues/193#issuecomment-636697897

### CODE REVIEW

File: src/Smalot/PdfParser/Config.php [hunk 1]
```diff
@@ -40,6 +40,8 @@ class Config
 {
     private $fontSpaceLimit = -50;
 
+    private $horizontalOffset = ' ';
+
```
Comment (k00ni) [inline src/Smalot/PdfParser/Config.php:44]: Which type is it? string...? Please add `@var` PHPDoc comment.

File: src/Smalot/PdfParser/Config.php [hunk 2]
```diff
@@ -78,6 +80,16 @@ public function setFontSpaceLimit($value)
         $this->fontSpaceLimit = $value;
     }
 
+    public function getHorizontalOffset()
+    {
+        return $this->horizontalOffset;
+    }
+
+    public function setHorizontalOffset($value)
```
Comment (k00ni) [inline src/Smalot/PdfParser/Config.php:88]: Based on the type of the property, use appropriate parameter / return value.

File: src/Smalot/PdfParser/PDFObject.php [hunk 3]
```diff
@@ -280,8 +280,7 @@ public function getText(?Page $page = null): string
                                 $current_position_td['x']
                             )
                         ) {
-                            // horizontal offset
-                            $text .= ' ';
+                            $text .= $this->config->getHorizontalOffset();
```
Comment (k00ni) [inline src/Smalot/PdfParser/PDFObject.php:283]: Can you construct a basic test which includes this change?

**Review by k00ni — CHANGES_REQUESTED** [review]
Thank you!

**Comment by k00ni** [comment]
Does this fix #193?

**Review by k00ni — APPROVED** [review]
Thank you for your changes, it looks good to me.

If you have the time can you tell me if this PR fixes #193 and #72?

Any remarks @j0k3r?

**Comment by rubenvanerk** [comment]
As I don't have access to the pdf's in those issues, I can't say for sure. But I think it does.

Since I used the pdf of #494, I can say that it fixes _that_ issue.

**Review by j0k3r — APPROVED** [review]

**Pull request merged**